cluster/images/kubemark/Makefile

```diff
@@ -15,8 +15,21 @@
 # build Kubemark image from currently built binaries containing both 'real' master and Hollow Node.
 # This makefile assumes that the kubemark binary is present in this directory.
 
+PROJECT?="google_containers"
+REGISTRY?="gcr.io"
+
 all:
-	docker build -t gcr.io/$(PROJECT)/kubemark .
+	gcloudpush
+
+build:
+	docker build -t $(REGISTRY)/$(PROJECT)/kubemark .
```

Comment:
@rthallisey In general, does this pattern have to be followed for all docker registries or can it differ across providers?